SimPEG/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -65,27 +83,10 @@ def __init__(
 
     _Jmatrix = None
     _pred = None
-    _scale = None
 
     def fields(self, m):
         if self.verbose:
             print(">> Compute DC fields")
-        if self._f is None:
```

Comment:
I think you need to leave these 3 removed lines in (73-76) as it will never set `self._f` if it was `None` and the user repeatedly calls `fields` (you can theoretically call `fields` without ever calling anything else).

Reply:
good catch. fixed